### PR TITLE
Implementing web-share API

### PR DESCRIPTION
**Pull request by insane-22**
I've tried adding the web-share api but completely unsure if it works or not. Any suggestions would be highly appreciated

### CODE REVIEW

File: src/components/Celebration.css [hunk 1]
```diff
@@ -7,26 +43,31 @@
...
-    0%{
+    0% {
         opacity: 0;
-    }100%{
+    }
+    100% {
         opacity: 1;
     }
-    
+>>>>>>> Stashed changes
```
Comment (idna001) [inline src/components/Celebration.css:72]:
```suggestion
>>>>>>> Stashed changes
```

File: src/components/Celebration.css [hunk 2]
```diff
@@ -1,4 +1,40 @@
-.celebration-notification{
+.celebration-notification {
+<<<<<<< Updated upstream
```
Comment (idna001) [inline src/components/Celebration.css:2]:
```suggestion

```

**Comment by idna001** [comment]
@insane-22 I inspect the behavior. 
The goal is to open the browsers sharing options, if supported 
Otherwise, it would be a default sharing popup. 

I couldn't fix this till now, but I will have a look again how navigator.share is working. 

In dev mode it's working for the default popup, but on preview URL not. I also will look this up.

**Comment by idna001** [comment]
Found out that there is a different way in react: 
https://www.brannen.dev/posts/using-the-web-share-api-in-react

**Comment by idna001** [comment]
Now it shares successfully if the API is supported. 
Open to-dos: 

- [ ] Add image Screenshot on sharing
- [ ] Add default if API is not supported

**Comment by insane-22** [comment]
Hey @idna001 , I greatly appreciate your invaluable inputs and guidance. Tried completing the tasks, kindly check:-
Demo video-
[Screencast from 13-10-23 02:30:17 AM IST.webm](https://github.com/idna001/Couple-Memory/assets/116253210/17c86c69-e101-49f3-9a5b-77d4518f3880)

**Comment by idna001** [comment]
@insane-22 
Cool, but this is not the complete behavior that I mean. 
In this case, displayed in your video, it should be displaying this view: 
grafik

Default View if sharing is not supported


If it is supported, the system view
For example Windows: 

grafik

**Comment by insane-22** [comment]
Oh ok @idna001 , I got it now, have made the changes, do see:-
[Screencast from 13-10-23 12:55:50 PM IST.webm](https://github.com/idna001/Couple-Memory/assets/116253210/d44f8ed4-7a2a-4c14-afdb-c08c893f058d)

**Pull request merged**